I/igraph/build_tarballs.jl

```diff
@@ -0,0 +1,80 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "igraph"
+version = v"0.10.15"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/igraph/igraph.git", "635b432eff0a89580ac9bb98068d2fbc8ef374f2")
+]
```

Comment:
You _can_ do this, but we do not recommend compiling from the git repo directly.  If you do so, you will need more tools, such as `flex` and `bison`.

We recommend using the official source archive, which has the parsers pre-generated and the documentation included: https://github.com/igraph/igraph/releases/download/0.10.15/igraph-0.10.15.tar.gz

Reply:
noted! Maybe this is related to why I was having some failures with/without flags you describe as unnecessary above